src/app/core/services/helper.service.ts

```diff
@@ -197,7 +197,7 @@ export class HelperService {
     const param = new HttpParams()
       .set(paramName, paramValue);
     console.log(param.toString());
-    return `${url}?${param.toString()}`;
+    return `${url.split('?')[0]}?${param.toString()}`;
```

Comment:
although we don't support multiple url parameters here yet, updating the url parameter is supported with this logic